tachyon/math/polynomials/univariate/univariate_evaluations_ops.h

```diff
@@ -56,6 +56,24 @@ class UnivariateEvaluationsOp {
     return self;
   }
 
+  static Poly Sub(const Poly& self, const Poly& other) {
+    const std::vector<F>& l_evaluations = self.evaluations_;
+    const std::vector<F>& r_evaluations = other.evaluations_;
+    if (l_evaluations.empty()) {
+      // 0 - g(x)
+      return -other;
+    }
+    if (r_evaluations.empty()) {
+      // f(x) - 0
+      return self;
+    }
```

Comment:
```suggestion
    }
  CHECK_EQ(l_evaluations.size(), r_evaluations.size());
```

Reply:
I don't think we need check for evaluations.

Reply:
@TomTaehoonKim It just tries to not open thread context!

Reply:
@Insun35 I'll add the size checks to everywhere cuz @ashjeong already mentioned about it.